src/tim/vx/ops/conv2d.cc

```diff
@@ -96,6 +96,34 @@ const std::vector<std::shared_ptr<Tensor>> Conv2d::ConstantInputsTensor() const
   }
 }
 
+// Handle float16 bias if clang compiler is no less than 15.0.0 version
+#if  __clang__ && __clang_major__ >=15
```

Comment:
#if defined(__clang__) && (__clang_major__ >= 15)

Reply:
Modified and added marco TIM_VX_OPS_CONV2D_WITH_F16BIAS under this condition